order.go

```diff
@@ -95,6 +95,31 @@ func (c *Client) CaptureOrder(orderID string, captureOrderRequest CaptureOrderRe
 	return capture, nil
 }
 
+// CaptureOrder with idempotency - https://developer.paypal.com/docs/api/orders/v2/#orders_capture
+// Endpoint: POST /v2/checkout/orders/ID/capture
+// https://developer.paypal.com/docs/api/reference/api-requests/#http-request-headers
+func (c *Client) CaptureIdempotentOrder(
+	orderID string,
+	captureOrderRequest CaptureOrderRequest,
+	requestID string,
+) (*CaptureOrderResponse, error) {
+	capture := &CaptureOrderResponse{}
```

Comment:
thank you for the PR. Could you please make the `CaptureOrder` method above use this method? Seems like the only difference is setting the `PayPal-Request-Id` header?

Reply:
sure!